style.css

```
@@ -0,0 +1,219 @@
+@import url("https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable.min.css");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --main-color: #3692ff;
+  --text-color: #374151;
+  --background-color: #cfe5ff;
+  --white-color: #ffffff;
+  --black-color: #111827;
+}
+
+body {
+  font-family: "Pretendard Variable", Pretendard, -apple-system,
+    BlinkMacSystemFont, system-ui, Roboto, "Helvetica Neue", "Segoe UI",
+    "Apple SD Gothic Neo", "Noto Sans KR", "Malgun Gothic", "Apple Color Emoji",
+    "Segoe UI Emoji", "Segoe UI Symbol", sans-serif;
+}
+
+button {
+  background-color: var(--main-color);
+  color: var(--white-color);
+  border: none;
+  cursor: pointer;
+  outline: inherit;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+ul,
+ol {
+  list-style: none;
+}
+
+input::focus {
+  outline: none;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 200px 10px;
+}
+
+h1 {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 56px;
+  color: var(--text-color);
+}
+
+h3 {
+  font-size: 24px;
+  font-weight: 500;
+  line-height: 29px;
+  letter-spacing: 0.08em;
+  color: var(--text-color);
+}
+
+/* Header */
+.login_button {
+  width: 128px;
+  height: 48px;
+  border-radius: 8px;
+  font-size: 16px;
+  font-weight: 600;
+  cursor: pointer;
+}
+
+/* Main - First Section */
+.banner {
+  background-color: var(--background-color);
+  overflow: hidden;
+}
+
+.banner_container {
+  position: relative;
+  display: flex;
+  align-items: center;
+  width: 1200px;
+  height: 540px;
+  margin: 0 auto;
+}
+
+.banner_img {
+  position: absolute;
+  left: 340px;
+  bottom: 0;
+}
+
+.banner_title {
+  margin-bottom: 32px;
+}
+
+.look_button {
+  width: 355px;
+  height: 56px;
+  border-radius: 40px;
+  font-size: 20px;
+  font-weight: 600;
+  line-height: 24px;
+  cursor: pointer;
+}
+
+/* Main - Second Section */
+.second_section {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  padding: 137px 0;
+  gap: 274px;
+}
+
+.info_content {
+  display: flex;
+  align-items: center;
+  gap: 64px;
+  width: 1200px;
+}
+
+.info_content img {
+  width: 588px;
+  height: 444px;
+  border-radius: 12px;
+}
+
+.info_description {
+  display: flex;
+  flex-direction: column;
+}
+
+.reverse {
+  flex-direction: row-reverse;
+}
+
+.text_end {
+  text-align: end;
+}
+
+.info_badge {
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 25px;
+  color: var(--main-color);
+  margin-bottom: 12px;
+}
+
+.info-title {
```

클래스이름을 구조화하시려한게 좋습니다 👍 BEM방식이 생각나기도 하고요

-는 오타인거같네요

```suggestion
.info_title {
```